tests/logical_operators.rs

```diff
@@ -81,3 +81,25 @@ fn nesting() {
   evaluate("'' || '' || '' || '' || 'foo'", "foo");
   evaluate("'foo' && 'foo' && 'foo' && 'foo' && 'bar'", "bar");
 }
+
+#[test]
+fn empty_variadics_are_falsey() {
```

Comment:
We should also add test cases for comparing empty variadics with the empty string with`==`, `!=`, `=~`, and `!~`, since those are cases where I can imagine accidentally introducing a breaking change.